### ISSUE TITLE
Allow creation of initial addons

### ISSUE THREAD

**Issue by maciaszczykm** [kubermatic/kubermatic#6953]
**User Story** It would be good to be able to create initial addons similar to the initial machine deployments. It was mentioned in https://github.com/kubermatic/kubermatic/issues/2928 as a related issue. The UI issue is here https://github.com/kubermatic/dashboard/issues/1904.

**Acceptance criteria** The API accepts addons in the create cluster request, serializes them into annotation and then controller creates them once cluster is ready (just like machine deployments).

**Comment by maciaszczykm** [comment]
Closing as duplicate of #6000.
